json-everything.net/wwwroot/meta/data-2022

```diff
@@ -22,9 +22,9 @@
             "additionalProperties": {
                 "type": "string",
                 "oneOf": [
-                    { "format": "uri-reference" },
                     { "format": "json-pointer" },
-                    { "format": "relative-json-pointer" }
+                    { "format": "relative-json-pointer" },
+                    { "format": "uri-reference" }
```

Comment:
You can add `"pattern": "^#"` to restrict this to fragment-only URIs if you go with that suggestion I made earlier.

Reply:
Do we have a format for full (absolute) URI that could also include a JP fragment?  I'd have to split out `uri-reference` to support "JP fragment URI" and "full URI with (+) optional JP fragment" as separate items (which is fine, but I'd have to do it).

Reply:
I've split this to `iri` and `iri-reference`-with-`pattern`.

(I've also updated to use IRI instead of URI.)

Reply:
> Do we have a format for full (absolute) URI that could also include a JP fragment? 

@gregsdennis one of my many half-designed `format`-replacement vocabularies would be capable of this 🙃